README.md

```diff
@@ -41,6 +41,14 @@ python -m pip install --upgrade pip
 python -m pip install -e '.[tests]'
 ```
 
+### Install test dependencies
+In order to run the tests, you will need tox. Two of the tests also use skopeo.
+
+```bash
+pip install tox
+sudo dnf install skopeo
```

Comment:
we should be fixing the tests, not updating the instructions for wrongly written tests....